packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -402,14 +402,6 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
         return _resourceConfig;
     }
 
-    /// @notice An external setter for the resource config.
-    ///         In the future, this method may emit an event that the `op-node` picks up
-    ///         for when the resource config is changed.
```

Comment:
I'm unsure of the origins of this comment, but we may want to move it to the internal `_setResourceConfig` method to preserve it
```solidity
    ///         In the future, this method may emit an event that the `op-node` picks up
    ///         for when the resource config is changed.
```